src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1921,7 +1921,7 @@ def create_function(cmd, resource_group_name, name, storage_account, plan=None,
                     deployment_container_image_name=None, tags=None):
     # pylint: disable=too-many-statements, too-many-branches
     if deployment_source_url and deployment_local_git:
-        raise CLIError('usage error: --deployment-source-url <url> | --deployment-local-git')
+        raise CLIError('usage error: --deployment-source-url <url> --deployment-local-git')
```

Comment:
i think we should keep `|` unless these 2 arguments should be used together.

Reply:
Sorry that was by mistake - reverted